src/Models/Environment.php

```diff
@@ -889,4 +889,25 @@ private function settings($setting = null)
         $response = (array)$this->request()->request($path, ['method' => 'get',]);
         return $response['data']->$setting;
     }
+
+    /**
+     * Checks if the environment has the build step enabled by looking
+     * at the environment variables defined in pantheon.yml.
+     *
+     * @return bool
+     */
+    public function isBuildStepEnabled()
+    {
+        $path = sprintf(
+            'sites/%s/environments/%s/variables',
+            $this->getSite()->id,
+            $this->id
+        );
+        $options = ['method' => 'get',];
```

Comment:
If you're already here making other tweaks, can we remove the renegade comma?

Reply:
Renegade commas are idiomatic in PHP at the end of arrays.

Reply:
Interesting, so it is. This is new. It seems very diff-friendly for a multiline definition but I'm not sure I see the point since it's on one line.